Sources/Charcoal/Extensions/NumberFormatterExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+extension NumberFormatter {
+    static let formatterWithSeparator: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .currency
```

Comment:
I think we should use `.decimal`. I think the only difference is `.currency` has two decimals.

Reply:
Good point! Fixed.